js/packages/components/chat/ChatSettingsMemberDetail.tsx

```diff
@@ -64,7 +65,7 @@ export const ChatSettingsMemberDetail: ScreenFC<'Group.ChatSettingsMemberDetail'
 	React.useLayoutEffect(() => {
 		navigation.setOptions({
 			title: displayName,
-		})
+		} as Partial<StackNavigationEventMap>)
```

Comment:
I don't know why we have to cast this object now, try to remove it and run `make lint` (that check the typescript errors) to see if we really have an error

Reply:
`make lint` doesn't work without this line, it appeared on this PR I don't know why